src/components/SignUp/Personality.js

```diff
@@ -48,6 +49,20 @@ class Personality extends Component {
       Actions.refresh({ rightTitle: 'SAVE', onRight: this.sendRequest.bind(this) });
   }
 
+  componentWillReceiveProps(props) {
```

Comment:
@MechanicKim I have considered this for a while, and as you can see in this(https://github.com/sv-bootcamp/bridgeme-client/blob/master/src/components/EditProfile.js#L17) I extracted these common props for 4 component, but, when it comes to 'onBack' and 'onRight' props, that function body cannot be decided at parents component(which mean 'EditProfile.js'). So I concluded that part cannot be separated. Please share some thoughts about this if you have any idea. Thanks!